packages/nodes-base/nodes/ExecuteWorkflowTrigger/ExecuteWorkflowTrigger.node.ts

```diff
@@ -46,10 +51,129 @@ export class ExecuteWorkflowTrigger implements INodeType {
 				],
 				default: 'worklfow_call',
 			},
+			{
+				displayName: 'Input Fields',
+				name: FIELDS,
```

Comment:
Since this will be called `Workflow Inputs` in the final version, can we also change the `name` to `workflowInputs` to make it more readable?